scripts/utils/constructProposalCalldata.ts

```diff
@@ -91,5 +102,25 @@ function getTimelockCalldata(proposal: ExtendedAlphaProposal, proposalInfo: Temp
     salt
   ]);
 
-  return `Calldata: ${calldata}\nExecute Calldata: ${executeCalldata}`;
+  if (podConfig) {
+    const proposalId = calcProposalId(proposal.targets, proposal.values, combinedCalldatas, predecessor, salt);
+    const registerMetadataCalldata = metadataFuncFrag.encodeFunctionData('registerProposal', [
+      podConfig.id,
+      proposalId,
+      proposal.description
+    ]);
+    return `Calldata: ${calldata}\nMetadata Calldata: ${registerMetadataCalldata}\nExecute Calldata: ${executeCalldata}`;
+  } else {
+    return `Calldata: ${calldata}\nExecute Calldata: ${executeCalldata}`;
+  }
+}
+
+export function calcProposalId(
+  targets: string[],
+  values: BigNumber[],
+  payloads: string[],
+  predecessor: string,
+  salt: string
+): string {
+  return '0x123';
```

Comment:
I'm guessing this is a placeholder to fix errors; my go-to for this is just to explicitly throw an error, ie:

`throw new Error("Method not yet implemented")`

this way I don't accidentally use a placeholder'd function if I forget to finish it.